plugins/InspectionGadgets/src/inspectionDescriptions/MathRoundingWithIntArgument.html

```diff
@@ -0,0 +1,17 @@
+<html>
+<body>
+Reports calls to <code>round</code>, <code>ceil</code>,
+<code>floor</code>, <code>rint</code> methods for <code>Math</code> and <code>StrictMath</code> with <code>int</code> as the argument.
+<p>These methods could be called in case the argument is expected to be <code>long</code> or <code>double</code>, and it may have unexpected results.</p>
+<p>The inspection provides a fix that simplify such expressions (except <code>round</code>) to cast to <code>double</code>.</p>
+<p><b>Example:</b></p>
+<pre><code>
+  double d1 = Math.floor(2);
+</code></pre>
+<p>After the quick-fix is applied:</p>
+<pre><code>
+  double d1 = (double)(2);
+</code></pre>
+<!-- tooltip end -->
+</body>
```

Comment:
Please add:
```
<p><small>New in 2023.1</small>
```
so users can find the new inspections between the thousands of older ones.

Reply:
Added